dtos/reading.go

```diff
@@ -35,7 +35,8 @@ type BaseReading struct {
 }
 
 type SimpleReading struct {
-	Value *string `json:"value"`
+	IsNull bool   `json:"isNull,omitempty"` // indicate the reading value should be null in the JSON payload
```

Comment:
```suggestion
	isNull bool   `json:"isNull,omitempty"` // indicate the reading value should be null in the JSON payload
```
don't expose this field

is there any difficulty to move this field to a new NullReading struct?

Reply:
Updated and created the NullReading struct.
Is the NullReading only used for simple reading?

Reply:
No, it can be applied to any valueType
it will cause the value, binaryValue, and objectValue all `: null`